### PR TITLE
Add join tables

### PR DESCRIPTION
**Pull request by alexmedinasf**
In this project, I added some many-to-many relationships and wrote more complex queries to use them to answer questions.

[LINK OF THE SCREENSHOTS](https://imgur.com/a/Tp0DF6u)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -91,4 +91,49 @@ update animals
 set owner_id = (select id from owners where full_name = 'Dean Winchester')
 where name in  ('Angemon','Boarmon');
```
Comment (V-Blaze) [inline data.sql:93]: - You have done great so far :+1: :+1: But there are a few Data you missed inserting into the DB. :+1: :+1: 

> Insert the following data for vets:
Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.

- [x] You didn't write the code to insert these data above into the DB, and it's definitely going to affect the result of your queries. I'll suggest you add the data and re-run your queries. :+1: :100:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @alexmedinasf 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


## Highlights
- Queries are written correctly :heavy_check_mark: 
- Readme looks professional :heavy_check_mark: 
- Well documented code :heavy_check_mark: 

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (alexmedinasf) [reply data.sql:93]: Hey @V-Blaze , thank you for the review, I happened to forget to add the data to the .sql file but I added it to the DB I used on PgAdmin4, so the results were okay. Thank you again for the review!

**Review by V-Blaze — APPROVED** [review]
Hi @kifle23 , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights


## Highlights
- Correct data inserted into the database✔✔
- Queries are written correctly :heavy_check_mark: 
- Screenshots show correct results :heavy_check_mark: 
- Readme looks professional :heavy_check_mark: 
- Excellent communication with the reviewer :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alexmedinasf** [comment]
Thank you for reviewing and approving my project!!

**Pull request merged**